syntax = "proto2";

message DirectoryEntryMessage{
    required string name = 1;
    enum Type  {DIRTYPE =0; FILETYPE=1;}
    required Type type  = 2;
    optional string checksum = 3;
}
